translator.cjs

```diff
@@ -344,16 +369,14 @@ const rl = readline.createInterface({
   output: process.stdout,
 });
 
-rl.question(`Choose a mode:\n1. Automatic mode\n2. Manual mode\n`, (answer) => {
-  rl.question('Enter source language code: ', (srcLang) => {
-    srcLang === '' ? (srcLang = 'en') : (srcLang = srcLang);
-    rl.question('Enter target language code: ', (targetLang) => {
-      targetLang === ''
-        ? (targetLang = process.env.DEFAULT_TARGET_LANGUAGE || 'hu')
-        : (targetLang = targetLang);
-      const translator = new Translator(srcLang, targetLang);
-      translator.run('src');
-      rl.close();
-    });
+rl.question('Enter source language code: ', (srcLang) => {
+  srcLang === '' ? (srcLang = 'en') : (srcLang = srcLang);
+  rl.question('Enter target language code: ', (targetLang) => {
+    targetLang === ''
+      ? (targetLang = process.env.DEFAULT_TARGET_LANGUAGE || 'hu')
```

Comment:
[nits] Unnecessary

Reply:
No it won't be. I left it for myself like this. These I'll - and already I started to change in my repository - to interactive parts driven by the filestructure of the project and the configuration driven by the AI, with selectable options...